test/irb/test_command.rb

```diff
@@ -265,7 +265,7 @@ def execute(_arg)
     end
 
     def test_def_extend_command
-      IRB::Command._register_with_aliases(:foobar, FooBarCommand, [:fbalias, IRB::ExtendCommandBundle::OVERRIDE_ALL])
+      IRB::ExtendCommandBundle.def_extend_command(:foobar, FooBarCommand, nil, [:fbalias, IRB::Command::OVERRIDE_ALL])
```

Comment:
I incorrectly updated the method called in this test in the command extension PR, so I restored it here.